terraform/global/s3/main.tf

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+resource "random_id" "terraform_state_bucket_name" {
+  prefix      = "${var.deployed_env}-terraform-"
+  byte_length = 8
+}
+
+resource "aws_s3_bucket" "terraform_state" {
+  bucket = random_id.terraform_state_bucket_name.hex
```

Comment:
Instead of `random_id.terraform_state_bucket_name.hex` we usually do something like `app-state-${account_id}` thats the convention we have been using for buckets names for state

Reply:
Cool, I will update and follow that convention.